com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderCompilers.java

```diff
@@ -45,14 +45,19 @@ private ArrayList<ShaderDesc.Language> getPlatformShaderLanguages(ShaderDesc.Sha
                 case X86Win32:
                 case X86_64Win32:
                 case X86Linux:
-                case Arm64Linux:
                 case X86_64Linux: {
                     if (shaderType == ShaderDesc.ShaderType.SHADER_TYPE_COMPUTE) {
                         shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLSL_SM430);
                     } else {
                         shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLSL_SM140);
                     }
                 } break;
+                case Arm64Linux: {
+                    if (shaderType != ShaderDesc.ShaderType.SHADER_TYPE_COMPUTE) {
+                        shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM300);
+                        shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM100);
+                    }
+                } break;
```

Comment:
I think this is a bit too limiting.
Using this approach we cannot switch from gles to glsl.

I think we need another way of selecting the shader language. @Jhonnyg do you have any thoughts around this?

Reply:
Hmm hm, yes I think we need a different approach to this. I kinda feel like you should be able to configure your project to build with both ES and desktop OpenGL support + configure which shader versions to include. If for example you don't care about ES2/WebGL1 you shouldn't be forced to include them. As it stands right now, I don't have a solution for this, we need to design that setup a bit.